pkg/operator/operator_test.go

```diff
@@ -256,7 +260,7 @@ func getDeployment(args []string, image string, modifiers ...deploymentModifier)
 	if dep.Annotations == nil {
 		dep.Annotations = map[string]string{}
 	}
-	dep.Annotations["operator.openshift.io/pull-spec"] = image
+	addDeploymentHash(dep)
```

Comment:
Can you use helper func from library-go? https://github.com/openshift/library-go/blob/master/pkg/operator/resource/resourceapply/apps.go#L59

Reply:
This has been updated.